helpers/skills/doc-generate/SKILL.md

```
@@ -0,0 +1,173 @@
+   - Feed findings back to LLM with the generated content
+   - Ask LLM to fix the specific issues
+   - Re-validate
+   - Repeat up to **3 iterations**
+
+5. Track iteration count and findings per iteration
+
+## Step 5: Write output files
+
+Write generated modules to `workspace/generated-docs/`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)